README.md

```diff
@@ -639,6 +639,140 @@ if (idxResponse.isLoginSuccessful()) {
+Optional<RemediationOption> remediationOptionsOptional = Arrays.stream(remediationOptions)
+        .findFirst();
+RemediationOption remediationOption = remediationOptionsOptional.get();
+FormValue[] formValues = remediationOption.form();
+
+// send identifier
+idxResponse = client.identify(IdentifyRequestBuilder.builder()
+        .withIdentifier("{identifier}") // email
+        .withStateHandle(stateHandle)
+        .build());
+
+// get remediation options to go to the next step
+remediationOptions = idxResponse.remediation().remediationOptions();
+remediationOptionsOptional = Arrays.stream(remediationOptions)
+        .filter(x -> "select-authenticator-authenticate".equals(x.getName()))
+        .findFirst();
+remediationOption = remediationOptionsOptional.get();
+
+// get authenticator options
+Map<String, String> authenticatorOptions = remediationOption.getAuthenticatorOptions();
+log.info("Authenticator Options: {}", authenticatorOptions);
+
+// select password authenticator
+Authenticator phoneAuthenticator = new Authenticator();
+phoneAuthenticator.setId(authenticatorOptions.get("password"));
+phoneAuthenticator.setMethodType("password");
+
+// build password authenticator challenge request
+ChallengeRequest phoneAuthenticatorChallengeRequest = ChallengeRequestBuilder.builder()
+        .withAuthenticator(phoneAuthenticator)
+        .withStateHandle(stateHandle)
+        .build();
+idxResponse = remediationOption.proceed(client, phoneAuthenticatorChallengeRequest);
+
+// check remediation options to continue the flow
+remediationOptions = idxResponse.remediation().remediationOptions();
+remediationOptionsOptional = Arrays.stream(remediationOptions)
+        .filter(x -> "challenge-authenticator".equals(x.getName()))
+        .findFirst();
+remediationOption = remediationOptionsOptional.get();
+
+// answer password authenticator challenge
+Credentials credentials = new Credentials();
+credentials.setPasscode("{password}".toCharArray());
+
+// build answer password authenticator challenge request
+AnswerChallengeRequest phoneSmsCodeAuthenticatorAnswerChallengeRequest = AnswerChallengeRequestBuilder.builder()
+        .withStateHandle(stateHandle)
+        .withCredentials(credentials)
+        .build();
+idxResponse = remediationOption.proceed(client, phoneSmsCodeAuthenticatorAnswerChallengeRequest);
+
+// check remediation options to continue the flow
+remediationOptions = idxResponse.remediation().remediationOptions();
+remediationOptionsOptional = Arrays.stream(remediationOptions)
+        .filter(x -> "select-authenticator-authenticate".equals(x.getName()))
+        .findFirst();
+remediationOption = remediationOptionsOptional.get();
+
+// get authenticator options
+authenticatorOptions = remediationOption.getAuthenticatorOptions();
+log.info("Authenticator Options: {}", authenticatorOptions);
+
+// select webauthn (fingerprint) authenticator
+Authenticator fingerPrintAuthenticator = new Authenticator();
```

Comment:
Technically webauthn can be any of the biometric/security authenticators like yubikey, face scan etc. It's better to rename `fingerPrintAuthenticator` as `webauthnAuthenticator` instead. 
That brings back the question whether the fields `authenticatorData` etc are specific to fingerprint only. But that's a different question

Reply:
good question, let me see if I can find an answer.